app/utils/check-username.js

```diff
@@ -0,0 +1,24 @@
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default async function checkUserName(userName) {
+  try {
+    const lowerCaseUsername = userName.toLowerCase();
+    const response = await fetch(
```

Comment:
same as above.

Reply:
I didn't get what you are suggesting here

Reply:
`We have an ember-data then why make API calls from the route file? Can we please shift this to service.js file of this route. This little steps will help us use ember-data to its full power in near future.` 

I meant the same comment there as above